reascripts/ReaSpeech/source/ReaSpeechUI.lua

```diff
@@ -183,6 +191,30 @@ function ReaSpeechUI:new_jobs(jobs, endpoint, callback)
   table.insert(self.requests, request)
 end
 
+function ReaSpeechUI.link(text, onclick, text_color, underline_color)
```

Comment:
this is one of the things i'd considered moving under the general organizational bucket of a "widget" but hadn't pulled the trigger. not necessarily suggesting it here, just a quick callout to bump it up in my mental cache.